src/__tests__/useTracking.test.js

```diff
@@ -1,34 +1,18 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import { mount } from 'enzyme';
 import React from 'react';
-import { renderToString } from 'react-dom/server';
-import track from '../withTrackingComponentDecorator';
 import useTracking from '../useTracking';
 
 describe('useTracking', () => {
-  it('throws error if tracking context not present', () => {
```

Comment:
Removed this test because calling `useTracking` without the tree being wrapped in `track` no longer throws an error.

Reply:
Makes sense -- but maybe we should restore this test and just assert that it now _doesn't_ throw?